railties/lib/rails/generators/rails/app/templates/Dockerfile.tt

```diff
@@ -9,6 +9,8 @@ WORKDIR /rails
 
 # Set production environment
 ENV RAILS_ENV="production" \
+    BUNDLE_DEPLOYMENT="1" \
```

Comment:
All those env variables are deprecated in bundler though, so maybe we should be using `bundle config`?

Reply:
I'm aware of _flags_ on `bundle install` being deprecated, but unaware of environment variables being deprecated.  `bundle config` will require some thought: configs default to per user and the build process is done under the root user and the deploy is done with rails.  Also apps may have local config.

Reply:
Yeah, what Sam said. AFAIK the variables are not deprecated, and what `bundle config` does it basically just to store some kind of `env` file.

Reply:
Ah, right, thanks for clarifying.